### DISCUSSION TITLE
Content Security Policy

### DISCUSSION THREAD

**Discussion by lbodev** [mercadopago/sdk-js#188] [General]
Hello, I am having difficulty implementing the Content Security Policy using the PHP SDK.
I would like some help on this point.

My .htaccess looks like this:

`# Content-Security-Policy (CSP)
Header set Content-Security-Policy 
   "default-src 'self'; 
   script-src 'report-sample' 'self' 'unsafe-inline' 'unsafe-eval' https://http2.mlstatic.com https://fonts.googleapis.com https://unpkg.com https://*.google.com https://*.gstatic.com; 
   script-src-elem 'self' 'unsafe-inline' https://cdn.datatables.net https://fonts.googleapis.com https://unpkg.com https://*.google.com https://*.gstatic.com; 
   style-src 'self' 'unsafe-inline' https://cdn.datatables.net fonts.googleapis; 
   style-src-elem 'self' 'unsafe-inline' https://cdn.datatables.net https://fonts.googleapis.com https://unpkg.com; 
   font-src 'self' https://fonts.gstatic.com; 
   object-src 'none'; frame-src 'self' https://*.google.com; 
   img-src 'self' data: http://*.w3.org https://api.qrserver.com https://img.youtube.com; 
   connect-src 'self' https://cdn.datatables.net;"`

And the error reported as shown in the image

![Content Security Policy Mercado Pago](https://github.com/mercadopago/sdk-js/assets/118546221/e64d5a2c-6cc2-4900-a872-1e88eb111ba7)

**Comment by lucmantovani** [comment]
Hello @lbodev!
The error is `Report Only` and shouldn't prevent from continuing with the payment
Could you finish the payment process?
